File: PluginMessagingUtils/src/main/java/org/openredstone/executors/RawExecutor.java
package org.openredstone.executors;

import org.bukkit.plugin.Plugin;
import org.openredstone.messages.ActionMessage;

public class RawExecutor extends Executor {

    public RawExecutor(Plugin plugin) {
        super(plugin);
    }

    @Override
    public void execute(ActionMessage actionMessage) {
        plugin.getServer().dispatchCommand(plugin.getServer().getConsoleSender(), String.join(" ", actionMessage.getArguments()));
    }
}

File: PluginMessagingUtils/src/main/kotlin/org/openredstone/executors/RawExecutor.kt
package org.openredstone.executors

import org.bukkit.plugin.Plugin
import org.openredstone.messages.ActionMessage

class RawExecutor(plugin: Plugin) : Executor(plugin) {
    override fun execute(actionMessage: ActionMessage) {
        plugin.server.dispatchCommand(
            plugin.server.consoleSender,
            actionMessage.arguments.joinToString(" ")
        )
    }
}
